src/main/java/com/xiaomi/infra/pegasus/rpc/async/ClientRequestRound.java

```diff
@@ -34,12 +34,15 @@ public ClientRequestRound(
       Table.ClientOPCallback cb,
       boolean enableCounter,
       long timeoutInMilliseconds) {
-    this(
-        op,
-        cb,
-        enableCounter,
-        System.nanoTime() + timeoutInMilliseconds * 1000000L,
-        timeoutInMilliseconds);
+    this.operator = op;
+    this.callback = cb;
+    this.timeoutMs = timeoutInMilliseconds;
+
+    this.enableCounter = enableCounter;
+    this.createNanoTime = System.nanoTime();
```

Comment:
`createNanoTime` was deleted in https://github.com/XiaoMi/pegasus-java-client/pull/105, but it is useful in this pr.